manager/container.go

```diff
@@ -356,7 +372,7 @@ func (self *containerData) nextHousekeeping(lastHousekeeping time.Time) time.Tim
 		}
 	}
 
-	return lastHousekeeping.Add(self.housekeepingInterval)
+	return lastHousekeeping.Add(jitter(self.housekeepingInterval, JitterFactor))
```

Comment:
I think your initial starting point should also have jitter in the case of initial burst of containers.